beta-20161027.md

```diff
@@ -0,0 +1,81 @@
+- `EXPLAIN` can now be used with `SHOW`, `HELP`, and `ALTER TABLE ... SPLIT`. [#10013](https://github.com/cockroachdb/cockroach/pull/10013)
+- The `current_schemas()` function is now supported, improving compatibility with ActiveRecord. [#9604](https://github.com/cockroachdb/cockroach/pull/9604)
+- The `SHOW ALL` command is now supported. [#10195](https://github.com/cockroachdb/cockroach/pull/10195)
+- The `DROP INDEX` and `ALTER INDEX` commands can now use unqualified index names. [#10091](https://github.com/cockroachdb/cockroach/pull/10091)
+
+### Command-Line Interface Changes
+
+- The `--host` argument now defaults to `localhost` in both secure and insecure modes. [#10134](https://github.com/cockroachdb/cockroach/pull/10134)
+- Improved error messages when a server is down or certificates are not configured correctly. [#9917](https://github.com/cockroachdb/cockroach/pull/9917)
+- The `--key` argument can now be passed via the environment variable `COCKROACH_KEY`, matching other similar flags. [#10092](https://github.com/cockroachdb/cockroach/pull/10092)
+
+### Performance Improvements
+
+- RocksDB bloom filters now use 10x less memory, with a negligible reduction in speed. [#10085](https://github.com/cockroachdb/cockroach/pull/10085)
+- Old time-series data is now garbage collected. [#9959](https://github.com/cockroachdb/cockroach/pull/9959)
+- Reduced overhead of the internal time-series metric system. [#9889](https://github.com/cockroachdb/cockroach/pull/9889)
+- Reduced impact of schema changes on regular queries. [#9798](https://github.com/cockroachdb/cockroach/pull/9798)
+- Schema changes now save their progress and can resume from their last checkpoint instead of from the beginning in the event of a node failure. [#10036](https://github.com/cockroachdb/cockroach/pull/10036)
+
+### Bug Fixes
+
+- `SET TIME ZONE 0` now sets the session time zone to UTC (other numbers already worked). [#9992](https://github.com/cockroachdb/cockroach/pull/9992)
+- `SHOW` commands for session variables now work correctly when prepared and executed separately. [#10013](https://github.com/cockroachdb/cockroach/pull/10013)
+- Columns that are a part of the primary key can now be renamed. [#10018](https://github.com/cockroachdb/cockroach/pull/10018)
+- Fixed a panic during transaction rollback. [#9961](https://github.com/cockroachdb/cockroach/pull/9961)
+- Float comparisons involving `NaN` are now correct in all cases. [#10112](https://github.com/cockroachdb/cockroach/pull/10112)
+- Clock offset monitoring is now more sensitive. [#10185](https://github.com/cockroachdb/cockroach/pull/10185)
+- A server that is stopped and immediately restarted can no longer use its old lease and lead to inconsistencies. [#9504](https://github.com/cockroachdb/cockroach/pull/9504)
```

Comment:
Sorry, got this confused with your newer PR. I think I'll just delete this completely since it's not really possible to describe it in user-friendly terms.